tests/playwright/page-object-model/shared/cookie-banner-component.ts

```diff
@@ -0,0 +1,41 @@
+import { Locator, Page, expect } from '@playwright/test'
+
+export class CookieBannerComponent {
+  readonly expect: CookieBannerNavigationComponentAssertions
+  readonly locator: Locator
+  readonly acceptCookiesLocator: Locator
+  readonly rejectCookiesLocator: Locator
+  readonly cookiePageLinkLocator: Locator
+
+  constructor (readonly page: Page) {
+    this.expect = new CookieBannerNavigationComponentAssertions(this)
+    this.acceptCookiesLocator = page.locator('[data-test="cookie-banner-accept"]')
```

Comment:
I would prefer not to use test ids unless necessary, we should be able to get these locators without them